### PR TITLE
DOM: document nullable return types & properties

### PR DESCRIPTION
**Pull request by BenMorel**
This is in line with other stubs.

### CODE REVIEW

File: dom/dom_c.php [hunk 1]
```diff
@@ -574,12 +574,12 @@ class DOMDocument extends DOMNode  {
...
 
     /**
-     * @var string
+     * @var string|null
      * @since 5.0
      * The location of the document or NULL if undefined.
      * @link https://php.net/manual/class.domdocument.php#domdocument.props.documenturi
      */
-    public $documentURI ;
+    public $documentURI;
```
Comment (AlexMovsesov) [inline dom/dom_c.php]:
```suggestion
    public $documentURI ;
```
This formatting change is unrelated to the fix

File: dom/dom_c.php [hunk 2]
```diff
@@ -885,7 +885,7 @@ public function getElementsByTagNameNS ($namespaceURI, $localName) {}
      * @param string $elementId <p>
      * The unique id value for an element.
      * </p>
-     * @return DOMElement the DOMElement or &null; if the element is
+     * @return DOMElement|null The DOMElement or &null; if the element is
```
Comment (AlexMovsesov) [inline dom/dom_c.php]:
```suggestion
     * @return DOMElement|null the DOMElement or &null; if the element is
```

File: dom/dom_c.php [hunk 3]
```diff
@@ -1520,7 +1520,7 @@ public function getAttributeNode ($name) {}
      * @param DOMAttr $attr <p>
      * The attribute node.
      * </p>
-     * @return DOMAttr old node if the attribute has been replaced or &null;.
+     * @return DOMAttr|null Old node if the attribute has been replaced or &null;.
```
Comment (AlexMovsesov) [inline dom/dom_c.php]:
```suggestion
     * @return DOMAttr|null old node if the attribute has been replaced or &null;.
```

File: dom/dom_c.php (hunk 1 above)
Comment (BenMorel) [reply dom/dom_c.php]: I took the opportunity to fix this (in addition to the 2 formatting changes below) because they were on the same line, and make the code more consistent with the rest of the stubs. If someone from JetBrains wants to keep formatting changes in another PR, I'll happily accept your changes, otherwise I think it's doing its bit to improve the stub quality.

Comment (AlexMovsesov) [reply dom/dom_c.php]: Ok, let's leave it as it is, thx for your contribution. I suggest to add nullability to the following properties: DomNode.parentNode (with corresponding documentation update), DomNode.firstChild, DomNode.lastChild, DomNode.previousSibling, DomNode.nextSibling and the same properties for the DOMElement

Comment (BenMorel) [reply dom/dom_c.php]: Very good point! I updated these properties in `DOMNode` + `DOMElement`, and also for [`DOMNode::$ownerDocument`](https://www.php.net/manual/en/class.domnode.php#domnode.props.ownerdocument). @AlexMovsesov do you work for JetBrains?

Comment (AlexMovsesov) [reply dom/dom_c.php]: Thx, yes, I am working in JetBrains
